src/object_store/src/object/opendal_engine/gcs.rs

```diff
@@ -28,6 +30,8 @@ impl OpendalObjectStore {
 
         builder.root(&root);
 
+        builder.write_fixed_size(GCS_BUFFER_SIZE);
```

Comment:
any context or benchmark ?

Reply:
We noticed that the part size of streaming upload for GCS is default to 8MB in the opendal implementation, which is different from the part size (16MB) we are uing for AWS S3. We decided to keep them as the same for consistency.

Reply:
yes, just to keep consistency, you can refer to https://cloud.google.com/storage/pricing?hl=zh-cn for the price.